tripper/__init__.py

```diff
@@ -3,7 +3,9 @@
 
```

Comment:
A general question here.
Do we _really_ need to import all of this from the base module? What's the benefit? Is there something that can be left out that isn't truly necessary to expose completely or can be relegated to sub-module imports instead (like, e.g. `en`)?

Reply:
Agree, maybe I have included too much. But I think that the namespaces and the three core classes Triplestore, Literal and Namespace should at least be in the __init__.py file. 

The errors are are well located in errors.py
If the utility functions, en(), parse_literal() and split_iri() should not be imported by __init__.py, I think they should be moved into a new module called utils.py.

Reply:
> (...) If the utility functions, en(), parse_literal() and split_iri() should not be imported by **init**.py, I think they should be moved into a new module called utils.py.

This is a great idea. They should probably have been put into such a file even if the goal _wasn't_ to get a dedicated sub-module for imports, but just to separate concerns and clean up the `triplestore.py` file further.

Reply:
done